source/Calamari.Tests/Fixtures/StructuredVariables/Approved/JsonFormatVariableReplacerFixture.ShouldReplaceWholeObjectWithStructure.approved.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
Don't think we need this file - couldn't see a method that maps up to the approval file. Perhaps created, then changed method name, and re-created a new one?

Reply:
Good catch. I changed approach halfway through and didn't mean to commit that.